app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id, class_name: 'Post'
+  has_many :comments, foreign_key: :author_id, class_name: 'Comment'
+  has_many :likes, foreign_key: :author_id, class_name: 'Like'
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
```

Comment:
Please ensure that your `application controller` has `current user` set to `User.first` according to the project requirement.
Example below:

```ruby
# app/controllers/application_controller.rb
...

def current_user
  User.first
end
```